topi/python/topi/x86/conv2d_avx_1x1.py

```diff
@@ -34,7 +36,11 @@ def _get_default_schedule(wkl, simd_width):
         if out_width % ow_factor == 0:
             for oh_factor in range(out_height, 0, -1):
                 if out_height % oh_factor == 0 and ow_factor * oh_factor < 32:
-                    return AVXConv1x1Fwd(ic_bn, oc_bn, oh_factor, ow_factor)
+                    parallel_chunk = max(multiprocessing.cpu_count()//2, 1)
+                    parallel_axis = wkl.out_filter // oc_bn * out_height // oh_factor
+                    while parallel_chunk > 1 and parallel_axis % parallel_chunk > 0:
+                        parallel_chunk -= 1
+                    return AVXConv1x1Fwd(ic_bn, oc_bn, oh_factor, ow_factor, int(parallel_chunk))
```

Comment:
We can move this part out of default schedule, since it can also be used for customized schedules?